File: web/plugins/strip.js
import { WebPlugin } from "../engine/plugin.js";

export class StripPlugin extends WebPlugin {
    constructor() {
        super("空白を消す");
    }

    async execute(line, context) {
        const match = line.match(/^空白を消す\s+「(.+)」$/);
        if (!match) return;

        context.runtime.write(
            match[1].trim()
        );
    }
}

File: pc/plugins/strip.py
import re
from core.plugin import Plugin


class StripPlugin(Plugin):
    def translate(self, line):
        match = re.fullmatch(r'空白を消す\s+「(.+)」', line)
        if not match:
            return None

        value = match.group(1)
        return f"print({value!r}.strip())"